test/WebJobs.Extensions.Http.Tests/Helpers/Fakes/FakeActivator.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I can see us needing a lot of these helpers in different extensions and repos. Will work on publishing the test helpers to mock/test against WebJobs and have that published as a NuGet package we can consume